packages/devtools_app/lib/src/screens/inspector_v2/inspector_controller.dart

```diff
@@ -681,20 +596,16 @@ class InspectorController extends DisposableController
 
   void applyNewSelection(
     RemoteDiagnosticsNode? newSelection,
```

Comment:
nit: fits on one line

Reply:
I made the default value for the parameter `false`, but we can't remove it because these inspector service methods are shared with the legacy inspector.